skimage/_shared/utils.py

```diff
@@ -286,3 +286,41 @@ def convert_to_float(image, preserve_range):
     else:
         image = img_as_float(image)
     return image
+
+
+def _set_order(image_dtype, order):
```

Comment:
`_set_interpolation_order` would be more informative

Reply:
`_validate_interpolation_order` or `_coerce_interpolation_order` would be even better.